src/main/java/seedu/address/model/person/DeadlineList.java

```diff
@@ -46,6 +78,14 @@ public String listFormat() {
         return result.toString();
     }
 
+    /**
+     * Gives the size of this deadline list, considering only deadlines given by user.
+     */
+    public int size() {
+        return this.deadlines.size() == 1 && this.deadlines.get(0).equals(EMPTY_DEADLINE) ? 0
+                : this.deadlines.size();
+    }
```

Comment:
I think this logic seems a bit weird. I'm not sure if I'm misunderstanding this, but can't we have `EMPTY_DEADLINE` return 0 on `emptyDeadline#size`?